src/lambdas/archive-user-logs/src/index.ts

```diff
@@ -0,0 +1,34 @@
+import { gunzip } from "zlib"
+import { promisify } from "util"
+import type { Event } from "./types"
+import { ValidEvent, ValidInput } from "./types"
+import { uploadToS3 } from "./utils/uploadToS3"
+
+export const handler = async (input: unknown): Promise<void> => {
+  try {
```

Comment:
We are throwing the error in this function anyway. This try/catch block seems to be redundant.